src/functionals.jl

```diff
@@ -18,6 +18,12 @@ function expectation(distr::DiscreteUnivariateDistribution, g::Function, epsilon
     sum(x -> f(x)*g(x), leftEnd:rightEnd)
 end
 
+function expectation(distr::CountableUnivariateDistribution,
+                     g::Function, epsilon::Real)
+    f = x->pdf(distr,x)
+    sum(x -> f(x)*g(x), support(distr))
```

Comment:
How about just
```
sum(support(distr)) do x
    pdf(distr,x) * g(x)
end
```

Reply:
Sure - I just copied from the existing code, but I'm happy to change it. Is there some reason to choose this?

Reply:
Not imensemsly it should all inline